netsblox.js

```diff
@@ -884,20 +856,25 @@ NetsBloxMorph.prototype.initializeCloud = function () {
 NetsBloxMorph.prototype.rawLoadCloudProject = function (project, isPublic) {
     var newRoom = project.RoomName,
         isNewRole = project.NewRole === 'true',
-        roleId = project.ProjectName;  // src proj name
+        roleName = project.ProjectName,
+        projectId = project.ProjectID;  // src proj name
```

Comment:
Maybe I should call this `RoomID`. I prefer `ProjectID` but this can be confusing since what the Snap code base considers a project is a role in netsblox and these instances haven't been renamed throughout the codebase.

Reply:
For now, I am going to leave it as ProjectID with the hopes that we can migrate the client terminology to match the standard netsblox terminology. However, this may change after we pull the latest from Snap (depending on the difficulty of the merge).